frontend/public/components/utils/_name-value-editor.scss

```diff
@@ -1,10 +1,24 @@
+.co-empty__header {
+  width: auto;
+}
+
 .pairs-list__row {
   padding-bottom: 15px;
 }
 
 .pairs-list__row-dragging {
-  padding-bottom: 15px;
-  opacity: 0
+  background-color: $color-pf-black-200;
+  border: 1px dashed $color-pf-black-300;
+  border-radius: 4px;
+  height: 28px;
+  margin-left: 28px;
+  margin-bottom: 15px;
+  overflow: hidden;
+  padding-right: 100px;
+  width: calc(83.34% + 11px);
```

Comment:
Might be good to add a comment explaining the math here (i.e., what's the significance of 83.34% and 11px?).